packages/core/src/hooks/useBlockMeta.ts

```diff
@@ -44,8 +44,16 @@ export function useBlockMeta(queryParams: QueryParams = {}) {
       }
   )
 
+  let timestamp: Date | undefined
+  try {
+    timestamp =
```

Comment:
Shouldn't we put this in `useMemo`? Won't it trigger re-renders if someone does:

```
const { timestamp } = useBlockMeta()

useEffect(xxx, [timestamp])
```
?

Reply:
Good catch